### PR TITLE
DNS setup - Changes to improve readability.

### PR DESCRIPTION
**Pull request by jhkrug**
There are 3 TODOs which need clarification/I need help on.

### CODE REVIEW

File: docs/installation/dns_setup.md [hunk 1]
```diff
@@ -1,157 +1,201 @@
...
+It is responsible for assigning IP addresses to services, that are load balanced, created on the cluster.
+Ingress controllers are such services (for example, Traefik) so should have an externally accessible IP address.
+This is needed for any Ingress resource to work.
+
+You can find the load balancer IP address of any service using `kubectl`. For example:
 
 ```
 kubectl get svc -n kube-system traefik -o jsonpath={@.status.loadBalancer.ingress}
 ```
+<!-- TODO: I would like a sample output from the previous command showing the IP address field.-->
```
Comment (andreas-kupries) [inline docs/installation/dns_setup.md:52]: On my local k3s this looks like
```
work@tagetarl:~/SUSE/dev/epinio> kubectl get svc -n kube-system traefik -o jsonpath={@.status.loadBalancer.ingress} ; echo
[map[ip:172.18.0.4]]
```

The `echo` in the command adds a newline for readability.
Note that this output is actually not JSON.
This is what go's `%+v` generates for the data structure.
As JSON it should have looked like
```
{ "ip": "172.18.0.4" }
```

File: docs/installation/dns_setup.md [hunk 2]
```diff
@@ -1,157 +1,201 @@
...
-Given you own the domain **"example.com"**, you would configure a subdomain e.g. **"test.example.com"** for Epinio.
-Now you will have to configure your DNS in a way, that any request towards "test.example.com" will resolve to the address found in the previous step.
+Assuming you own the domain `example.com`, you would configure a subdomain, for example, `test.example.com` for Epinio.
+You can now configure your DNS so that any request for `test.example.com`
+resolves to the address you obtained in the previous step.
 
-- test.example.com => "INGRESS-IP"
-- \*.test.example.com => "INGRESS-IP"
+<!--TODO: We have described what needs to be configured in DNS, but not actually shown how to do it.
+Do we need to? Or am I missing the point.-->
```
Comment (andreas-kupries) [inline docs/installation/dns_setup.md:65]: The commands/actions strongly depend on the exact DNS vendor used. I.e. `AWS Route 53` is different from whatever `Google Cloud` offers, is different from `MicroSoft Azur`, is different from ...

File: docs/installation/dns_setup.md [hunk 3]
```diff
@@ -1,157 +1,201 @@
...
 
-### AWS EKS and Route53
+We will use the [Amazon Route53](https://aws.amazon.com/route53/) to create a wildcard domain within an existing "Hosted zone", like `example.com`.
 
-As an example we will use the [AWS Service Route53](https://aws.amazon.com/route53/) to create a wildcard domain within one of your existing "Hosted zones", e.g. **example.com**.
-
-Given Epinio ingress installation provided you with the following hostname:
+Suppose an Epinio ingress installation has provided you with the following hostname:
+<!--TODO: Is the hostname below an actual format of an Amazon AWS hostname.
+If so that's fine, if not I think I will shorten it for a little more clarity.-->
```
Comment (andreas-kupries) [inline docs/installation/dns_setup.md:84]: At first glance that looks correct. @mmartin24 @thehejik might know better.

File: docs/installation/dns_setup.md (hunk 2 above)
Comment (andreas-kupries) [reply docs/installation/dns_setup.md:65]: And we do seem to show it for Route 53, Azure, and Bind, in the examples coming after.

File: docs/installation/dns_setup.md (hunk 1 above)
Comment (jhkrug) [reply docs/installation/dns_setup.md:52]: Thank you.

File: docs/installation/dns_setup.md (hunk 3 above)
Comment (jhkrug) [reply docs/installation/dns_setup.md:84]: Ok, I may exaplain and use something shorter anyway. Unless @mmartin24 or @thehejik have objections.

File: docs/installation/dns_setup.md [hunk 4]
```diff
@@ -1,157 +1,201 @@
...
-title: ""
+title: "DNS setup"
+description: How to setup DNS for Epinio and associated issues.
+keywords: [epinio, kubernetes, k8s, DNS]
 ---
 
-#  DNS setup
+During an [Epinio installation](install_epinio.md),
+a "system" domain must be specified in the `global.domain` helm field.
+This field is used to access an Epinio API server and to create default routes for the deployed applications.
```
Comment (thehejik) [inline docs/installation/dns_setup.md:11]: This value is used for creating an ingress resource for accessing Epinio API server and Epinio webUI...

File: docs/installation/dns_setup.md (hunk 3 above)
Comment (thehejik) [reply docs/installation/dns_setup.md:84]: Looks good to me, no objections, the hostname format matches with what we get from EKS.

File: docs/installation/dns_setup.md (hunk 4 above)
Comment (thehejik) [reply docs/installation/dns_setup.md:11]: Actually "Epinio API server and an Epinio WebUI" are using the same ingress named `epinio` (listening on **epinio.**`test.example.com`) but I believe it is worth to mention it. 

Now I noticed another ingress called `dex` (listening on **auth.**`test.example.com`) used for `Log in with Auth Provider` on Epinio login page.

I like the first option enhanced by dex:

During an Epinio installation a system domain must be specified in the 'global.domain' helm field. This field is used to create an ingress resource for access to an Epinio API server, an Epinio WebUI, a DeX and to create default routes for the deployed applications.

**Comment by jhkrug** [comment]
I'd appreciate a final review. I've changed quite a few things and would like a technical review to make sure it's still accurate. Thanks.

File: docs/installation/dns_setup.md [hunk 5]
```diff
@@ -1,157 +1,207 @@
...
-Given Epinio ingress installation provided you with the following hostname:
+If an Epinio ingress installation has provided you with the following hostname:
 
 ```bash
 Traefik Ingress info: [{"ip":"10.0.0.1"}]
 ```
 
-Now you will have to add two A records, for the subdomain, e.g. "test" to have "test.example.com", resp. "\*.test.example.com" to e.g. "/var/lib/named/master/forward/example.com".
-Replace "10.0.0.1" with the IP from "Traefik Ingress info", and "test.example.com" with your custom domain.
+You need to add two A records, for the subdomain and widlcard, so "test" to have `test.example.com`, and `*.test.example.com` to `/var/lib/named/master/forward/example.com`.
```
Comment (andreas-kupries) [inline docs/installation/dns_setup.md:187]:
```suggestion
You need to add two A records, for the subdomain and wildcard, so "test" to have `test.example.com`, and `*.test.example.com` to `/var/lib/named/master/forward/example.com`.
```

**Review by andreas-kupries — CHANGES_REQUESTED** [review]
small stuff
technical looks ok to me, defering to @thehejik's expertise however

**Review by andreas-kupries — APPROVED** [review]

**Pull request merged**